homeassistant/components/hue/v2/helpers.py

```diff
@@ -0,0 +1,21 @@
+"""Helper functions for Philips Hue v2."""
+
+
+def brightness_helper(brightness):
```

Comment:
maybe make it more clear what this method is about.
My suggestion would be to name the function "get_hue_brightness"

Reply:
Just an idea for the naming, may be we change it to "normalize_hue_brightness" ?

Reply:
Sounds good!